README.md

```diff
@@ -159,25 +159,26 @@ cd $(mktemp -d ${TMPDIR:-/tmp}/repro-XXXXXXX)
 # Create a dataset to contain mriqc output
 datalad create -d ds000003-qc -c text2git
 cd ds000003-qc
+# Install input data:
+datalad install -d . -s https://github.com/ReproNim/ds000003-demo sourcedata
 # Install our containers collection:
-datalad install -d . ///repronim/containers
+datalad install -d . -s ///repronim/containers code/containers
 # Optionally -- copy container of interest definition to the current (or desired)
 # version # to facilitate reproducibility while still being able to upgrade containers
 # subdataset if so desired to get access to newer versions.
 # We will also use 0.16.0 since newer ones require more memory and
 # would fail to run on CI.
-containers/scripts/freeze_versions --save-dataset=. bids-mriqc=0.16.0
-# Install input data:
-datalad install -d . -s https://github.com/ReproNim/ds000003-demo sourcedata
+datalad run -m "Downgrade/Freeze mriqc container version" \
+  code/containers/scripts/freeze_versions --save-dataset=. bids-mriqc=0.16.0
 # Setup git to ignore workdir to be used by pipelines
 echo "workdir/" > .gitignore && datalad save -m "Ignore workdir" .gitignore
 # Execute desired preprocessing while creating a provenance record
 # in git history
 datalad containers-run \
-        -n bids-mriqc \
+        --container-name code/containers/bids-mriqc \
         --input sourcedata \
         --output . \
-        '{inputs}' '{outputs}' participant group -w workdir
+        '{inputs}' '{outputs}' participant --participant-label sub-02 -w workdir
 )
```

Comment:
whatever you need to change, change in your step by step version, not here!

Reply:
The monolithic script as written doesn't run locally for me. I made this change hoping it would run/fail faster for development. And if it does work IMO this is more appropriate for a demo (no need to do all the compute right?)

I'll change it back